autoload/neoformat/formatters/erlang.vim

```diff
@@ -0,0 +1,11 @@
+function! neoformat#formatters#erlang#enabled() abort
+    return ['erlfmt']
+endfunction
+
+function! neoformat#formatters#erlang#erlfmt() abort
+    return {
+        \ 'exe': 'erlfmt',
+        \ 'args': ['erlfmt', "-"],
```

Comment:
I think you don't need the `erlfmt` in the args, otherwise I'm pretty sure neoformat will construct a command like `erlfmt erlfmt -`

```suggestion
        \ 'args': ["-"],
```

Reply:
Oh interesting, in my brief testing with my fork it seemed to have been working as is. But I'll try this suggestion and update

Reply:
Alright, that does appear to work as you suggested. I added eee4dbe